styles.css

```diff
@@ -629,3 +633,183 @@ textarea::placeholder {
   font-size: 15px;
   line-height: 24px;
 }
+
+.tonic-1 {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    max-width: 100%;
+  }
+
+  .dsk-header-logo {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 20px;
+    background-color: #fff;
+    padding-top: 1.875rem;
+  }
+
+  #body-cover {
+    background-image: url('images/Header\ bgDesktop.svg');
+    background-size: cover;
+    padding-bottom: 21rem;
+    border-left: 2rem solid white;
+    min-height: 30rem;
+    border-bottom: 6rem solid white;
+  }
+
+  .image-icon {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  .logo-2 {
+    margin-left: 10rem;
+  }
+
+  .intro {
+    margin-left: 16.25rem;
+    margin-right: 16.25rem;
+    max-width: 45.5rem;
+  }
+
+  .lets-connect {
+    margin-left: 16.25rem;
+  }
+
+  .social-buttons {
+    margin-left: 16.25rem;
+    margin-top: 1rem;
+  }
+
+  .nav-content {
+    list-style-type: none;
+    display: flex;
+    justify-content: space-between;
+    padding-right: 172px;
+    align-items: center;
+    gap: 86px;
+  }
+
+  .title {
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    margin-top: 9.438rem;
+    margin-left: 16.25rem;
+    margin-right: 16.25rem;
+  }
+
+  .works {
+    /* display: flex;
+    flex-direction: row; */
+    display: grid;
+    grid-template-columns: 0.5fr 1fr;
+    max-width: 75%;
+    margin-left: 15rem;
+  }
+
+  .works2 {
+    grid-template-columns: 0.5fr 1fr reverse;
+  }
+
+  .tonic-1 {
+    display: block;
+    padding: 24px 24px 24px;
+  }
+
+  .tonic {
+    display: none;
+  }
+
+  .header-tonic {
+    padding-top: 20px;
+  }
+
+  .stories {
+    padding-top: 20px;
+  }
+
+  .myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: auto;
+  }
+
+  .languages-below {
+    padding-top: 50px;
+    margin-left: 40px;
+  }
+
+  .language-icons {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
```

Comment:
- [ ] You did great on the `about-me` section, but here are some uggestions to make to give the design a better user exerince 👍 

- Firstly, there's a margin on the left on the Figma design which you need. without that, you can see how visually not o appealing it is. Kindly fix this 👍 
- Secondly, The button is placed on the same column and aligned with the short paragraph. Please adjust it to have a better user view which translates to better experience 👍 
- Lastly, there are three icons on the Figma design placed at the rear end of this section. I've attached comparison images for you to have a better view to make the review and changes required. Kindly adjust these to complement your work so far 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/222940237-eca50c0f-2179-4b43-9193-959bb1237110.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/222940253-84b9f35c-d129-44f5-8afe-1e3f204e210b.png)

Reply:
Well done on moving the arrows to their right positions 💯 but you missed out on two important requirements here:
- [ ] There should be another arrow pointing to the bottom on the extreme-right of `Languages` just like those you have for `Frameworks` and `Skills`, also make sure that your arrows are at the end of the lines just like in the Figma, this is what you have 👇🏽 
![Screenshot 2023-03-05 at 00 35 03](https://user-images.githubusercontent.com/45739963/222943493-8e3373da-c777-4e2d-b5b1-7d3d9fe771f6.png)
- [ ] Well done on aligning the content but you need to add some margin to the left 📏 to make it fit the design and also you need to align your social icons with the rest of the content. 😉 
![Screenshot 2023-03-05 at 00 42 08](https://user-images.githubusercontent.com/45739963/222943680-26c985d2-807d-4cc2-b2a4-9202c181a39f.png)